app/views/home/index.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1 class="text-center">Khan Academy Myanmar Community</h1>
-<p>This is the home page.</p>
+<div class="row">
+	<div class="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-xs-12" style="float:left;text-align:center;margin-top:100px;">
```

Comment:
Don't sue tab characters for indentation - they render differently on different viewers. Instead, use spaces.

For the div's class, you don't need to specify `col-xl` through `col-sm`, as each class scales up. Keep `col-xs`.

Declaring styles directly in HTML is not preferred to keeping them in CSS stylesheets. Bootstrap also offers classes to center elements. On the image element below, set its class as `img-responsive center-block`.